src/show/components/Region/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const regions = [
+  'Choose',
+  'United Kingdom',
+  'United States',
+];
+
+const Region = ({ inputs, handleInputChange, setInputs }) => {
+  return (
+    <div className="mt-3 textclr text-zinc-400">
+      <label className="font-medium block mb-1">Select Region</label>
+      <select
+        name="region"
+        value={inputs.region}
+        onChange={(e) => handleInputChange('region', e.target.value, setInputs)}
```

Comment:
create a global function if you have to do this again and again

Reply:
done